server.js

```diff
@@ -1,7 +1,17 @@
 const express = require('express')
 const path = require('path')
+const fs = require('fs')
 const formidable = require('formidable')
 const app = express()
+const PORT = process.env.PORT || 9000
+const UPLOAD_DIR = path.join(__dirname, 'uploads/')
+const CORS = process.env.NODE_ENV === 'production' ? process.env.HOST : '*'
+
```

Comment:
I added this in so you don't accidentally push this to production while allowing anyone to hit it.